modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/provider/test/DefaultSegmentsEntryProviderTest.java

```diff
@@ -237,6 +237,38 @@ public void testGetSegmentsEntryIdsWithContextCriterionAndDefaultUser()
 
 		Assert.assertArrayEquals(
 			new long[] {segmentsEntry.getSegmentsEntryId()}, segmentsEntryIds);
+
+		context = new Context();
+
+		context.put(Context.LANGUAGE_ID, "en");
+
+		segmentsEntryIds = _segmentsEntryProvider.getSegmentsEntryIds(
```

Comment:
Could you please create a new test for each use case? Namely, for every time the method under testing is called. This way it's difficult to say what scenario we're testing so that the failures might not be providing useful info